src/renderer/components/sideBar/search.vue

```diff
@@ -116,13 +117,23 @@
         'searchIncludeHidden': state => state.preferences.searchIncludeHidden,
         'searchNoIgnore': state => state.preferences.searchNoIgnore,
         'searchFollowSymlinks': state => state.preferences.searchFollowSymlinks
-      })
+      }),
+      searchResultInfo () {
+        const fileCount = this.searchResult.length
+        const matchCount = this.searchResult.reduce((acc, item) => {
+          return acc + item.matches.length
+        }, 0)
+
+        return `${matchCount} ${matchCount > 1 ? 'matches' : 'match'} in ${fileCount} ${fileCount > 1 ? 'files' : 'file'}`
+      }
     },
     methods: {
       search () {
         // No root directory is opened.
         if (!this.projectTree || !this.projectTree.pathname) {
-          return
+          return new Notification('No folder opened', {
```

Comment:
Why do you choose to use the native notification system instead of our own?

Reply:
There was some confusion in the use. The initial reason was that the notifications that appeared on the setup page were not very good, and the native notifications were used, which were later used elsewhere. In fact, our own notices also have some drawbacks. Confirmation and cancellation are not obvious, and it may be necessary to optimize later.